src/main/java/hudson/remoting/RemoteInvocationHandler.java

```diff
@@ -719,63 +718,63 @@ private void reportStats() {
             if (m15Std > 1 && 100 < m15Avg - 2 * m15Std) {
                 if (tStd > 1 && 100 < tAvg - 2 * tStd) {
                     logger.log(Level.SEVERE,
```

Comment:
These are some sort of exotic conditions with no test coverage that I know of, so just hoping it works.

Reply:
see #81